lib/Runtime/Base/ThreadContext.h

```diff
@@ -325,6 +325,10 @@ class ThreadConfiguration
         {
             EnableExperimentalFeatures();
             ResetExperimentalFeaturesFromConfig();
+
+#if TTD_VSCODE_WORK_AROUNDS
+            m_Simdjs = false;
+#endif
```

Comment:
Not needed anymore, we have COMPILE_DISABLE_Simdjs option to disable Simdjs in node scenario at compile time.